SocialFilter/sections/css_extended.txt

```diff
@@ -1,6 +1,7 @@
 !
 ! Extended CSS rules
 !
+indulgexpress.com#?#.facebook_timeline:upward(2)
```

Comment:
`indulgexpress.com##.facebook_timeline` is enough

Reply:
There is still an arrow button without upward.

Reply:
What arrow do you mean? Please make a screenshot

Reply:
Will you merge if I send the screenshot? ;)